crds/core.oam.dev_definitions.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: definitions.core.oam.dev
+spec:
+  group: core.oam.dev
+  names:
+    kind: Definition
+    listKind: DefinitionList
+    plural: definitions
+    shortNames:
+    - def
+    singular: definition
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.type
+      name: TYPE
+      type: string
```

Comment:
please also omit the timestamp.

Reply:
There is no `timestamp` in the spec, do you mean to omit the type?